### PR TITLE
Update lesson_check.py

### PR DESCRIPTION
**Pull request by bkmgit**
Enable use of a wider range of programming languages

Try to fix build error:

https://github.com/carpentries-incubator/hpc-intro/pull/341/checks?check_run_id=2373944669#

### CODE REVIEW

File: bin/lesson_check.py [hunk 1]
```diff
@@ -91,6 +91,17 @@
     'language-r',
     'language-shell',
     'language-sql',
+    'language-fortran',
```
Comment (tkphd) [inline bin/lesson_check.py:94]: Good additions. Please alphabetize the list.

**Review by tkphd — CHANGES_REQUESTED** [review]

File: bin/lesson_check.py (hunk 1 above)
Comment (bkmgit) [reply bin/lesson_check.py]: done, thanks for the suggestion

File: bin/lesson_check.py [hunk 2]
```diff
@@ -80,30 +80,30 @@
...
-    'language-scala',
-    'language-go',
-    'language-chapel',
-    'language-julia',
-    'language-rust',
-    'language-perl',
-    'language-lua',
-    'bash',
+    'output',
     'python'
```
Comment (tkphd) [inline bin/lesson_check.py:105]: Need a comma after 'python'

**Review by tkphd — CHANGES_REQUESTED** [review]

**Review by tkphd — APPROVED** [review]
Looks good to me! Thanks for the patches.

File: bin/lesson_check.py (hunk 2 above)
Comment (bkmgit) [reply bin/lesson_check.py]: Thanks.

**Comment by ocaisa** [comment]
Does this mean the theme can actually syntax highlight these languages? Did you test all these?

**Comment by tkphd** [comment]
Based on [`lesson.scss`](https://github.com/carpentries/styles/blob/gh-pages/assets/css/lesson.scss), these should render in colored boxes, but will not have the language name printed above until/unless they are all added.

**Comment by ocaisa** [comment]
Ok, but if they are all second class citizens in this context how do we make clear to anyone who wants to use them?

**Comment by tkphd** [comment]
* Is there a way to create an add-on SCSS file to provide complete coverage for our repos?
* The SCSS matches `language-*` to apply the colorbox, but explicitly writes labels for a handful of "supported" languages. Could the pattern matching be updated to both build the colorbox *and* use the `*` as the box title?
* The syntax highlighters, Rouge and Pygment, support all these languages and more, so the restriction is arbitrary.

**Comment by tkphd** [comment]
(implicitly, instead of warding off developers from using these syntax flags, I'm suggesting updating the upstream machinery so it "just works")

**Comment by bkmgit** [comment]
Can add these. Thanks for the suggestion.

**Comment by tkphd** [comment]
The Fortran code renders like this. "Code" is not untrue, and it looks like the syntax highlighter sorted out what to do. I don't think this requires a disclaimer, though the upstream improvements would still be welcome.

**Comment by tkphd** [comment]
@bkmgit, we lean on [carpentries/styles](https://github.com/carpentries/styles) for our CSS, since we really don't want to maintain the back-end at that level unless it's really necessary. Please don't add CSS to this repo, unless the above rendered code block is unacceptable.

**Comment by bkmgit** [comment]
Thanks for the suggestion -  https://github.com/carpentries/styles/pull/580

**Comment by tkphd** [comment]
:tada: @bkmgit's [PR](https://github.com/carpentries/styles/pull/580) was merged!

**Pull request merged**